src/routers/genericRouters/routerEdges.ts

```diff
@@ -333,16 +333,23 @@ export class RouterEdges extends RouterFilterBy {
     return await cursor.all()
   }
 
-  // A --(edge)--> B, (edge) --> C => given ID for C, return B
-  async getPrimaryTargetFromHyperEdgeByID (targetId: string, page: number = 0, sortBy: string = '', customSecondaryFilter: string = '', verbose: boolean = false): Promise<any[]> {
+  // A --(edge)--> B, (edge) --> C => given ID for C, return B and A
+  async getPrimaryTargetFromHyperEdgeByID (targetId: string, page: number = 0, sortBy: string = '', customSecondaryFilter: string = '', verbose: boolean = false, extraDataFrom: string = 'edge'): Promise<any[]> {
```

Comment:
these methods are returning a different format now and will impact the endpoints already using them

we should create new ones for ASB